Microsoft.Toolkit.Uwp.UI.Controls/Expander/Expander.cs

```diff
@@ -175,6 +180,40 @@ private void UpdateDisplayModeOrExpanderDirection(bool useTransitions = true)
             {
                 VisualStateManager.GoToState(this, visualState, useTransitions);
             }
+
+            UpdateOverlayAlignment();
+        }
+
+        private void UpdateOverlayAlignment()
```

Comment:
This is a lot of logic outside the template...

Can't we just move the `ContentOverlay` out of the rootgrid if it needs to be stretched differently?

Reply:
I don't think so. By design, the `RootGrid` is the only UIElement on top of others. This logic is required because we have two behaviors based on the `ContentOverlay` and it feels natural to put it inside the grid. The other alternative would be to duplicate the content in 2 different parts of the Template, but I think it's not a good option.